Moving getattr_from_fqn to torch.quantization.utils

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #63107

moving this function because the functionality would be useful outside of ns

Differential Revision: [D30260735](https://our.internmc.facebook.com/intern/diff/D30260735/)

HDCharles committed:
Moving getattr_from_fqn to torch.quantization.utils

moving this function because the functionality would be useful outside of ns

Differential Revision: [D30260735](https://our.internmc.facebook.com/intern/diff/D30260735/)

[ghstack-poisoned]

HDCharles committed:
Update on "Moving getattr_from_fqn to torch.quantization.utils"


moving this function because the functionality would be useful outside of ns

Differential Revision: [D30260735](https://our.internmc.facebook.com/intern/diff/D30260735/)

[ghstack-poisoned]

supriyar reviewed (approved):
lgtm, are all the formatting changes intentional in this PR?

HDCharles committed:
Update on "Moving getattr_from_fqn to torch.quantization.utils"


moving this function because the functionality would be useful outside of ns

Differential Revision: [D30260735](https://our.internmc.facebook.com/intern/diff/D30260735/)

[ghstack-poisoned]

HDCharles commented:
yeah, I was failing lint checks so I formatted doc
